src/Storefront/Resources/app/storefront/src/utility/modal-extension/ajax-modal-extension.util.js

```diff
@@ -15,13 +15,16 @@ const URL_DATA_ATTRIBUTE = 'data-url';
  *
  * Notice: The response template needs to have the markup as defined in the Bootstrap docs
  * https://getbootstrap.com/docs/4.3/components/modal/#live-demo
+ *
+ * @deprecated tag:v6.4.0 - Use UrlModalPlugin instead
  */
 export default class AjaxModalExtensionUtil {
 
     /**
      * Constructor.
      */
     constructor(modalBackdrop = true) {
+        console.log('Using the AjaxModalExtensionUtil is deprecated and will be removed in 6.5.0. Use UrlModalPlugin instead');
```

Comment:
```suggestion
        console.warn('Using the AjaxModalExtensionUtil is deprecated and will be removed in v6.4.0. Use the UrlModalPlugin instead');
```

Reply:
6.4.0 😲  🎉

Reply:
:man_shrugging: It's written above already :wink:

Reply:
I thought when it is deprecated in 6.4 it is still there but discouraged to use. So it has to be removed in 6.5. I still changed it though.

Reply:
Some projects might handle it this way, in this case the deprecation can be read as `@deprecated [. to be removed with] tag:v6.4.0`. This way developers are informed to use the new code as soon as it's merged, the old code gets removed with the next major version general.